Mathlib/Analysis/Analytic/Composition.lean

```diff
@@ -1033,7 +1024,7 @@ def sigmaCompositionAux (a : Composition n) (b : Composition a.length)
       (by
         rw [← a.blocks.join_splitWrtComposition b]
         exact mem_join_of_mem (List.get_mem _ _ _) hi)
-  blocks_sum := by simp only [Composition.blocksFun, nthLe_map', Composition.gather]; rfl
+  blocks_sum := by simp only [Composition.blocksFun, get_map, Composition.gather]
 #align composition.sigma_composition_aux Composition.sigmaCompositionAux
 
 -- porting note: this needs `Composition.blocksFun` to be refactored in order to remove `nthLe`
```

Comment:
You've now done this refactor, right? Can the `nthLe` be easily removed?

Reply:
Yeah, the `nthLe` could easily removed in all cases, but in `sizeUpTo_sizeUpTo_add`, where I didn't yet find the `get`-version of `List.sum_take_succ`, which introduces the `nthLe` there...
I will have another look.

Reply:
Changing `List.sum_take_succ` (and therefore `List.prod_take_succ`), will require some more change, so I propose doing this in #8043. So this pull request here is ready for review...